tidb-cloud/changefeed-sink-to-apache-kafka.md

```diff
@@ -65,17 +95,32 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
+      4. Fill the **Endpoint Service Name** which is configured in [Setup Self Hosted Kafka Private Link Service in AWS](/tidb-cloud/setup-self-hosted-kafka-pls.md)
+      5. Fill the **Boostrap Ports**, suggest at least one port for one AZ. You can use commas `,` to separate multiple ports.
+   3. If you select **Private Service Connect**
+      1. Make sure you fill the same unique ID in **Kafka Advertised Listener Pattern** when you [Setup Self Hosted Kafka Private Service Connect in Google Cloud](/tidb-cloud/setup-self-hosted-kafka-psc.md) in **Network** section.
+      2. Double-check the **Kafka Advertised Listener Pattern** by clicking the button **Check usage and generate**, which will show message to help you validate the unique ID.
+      3. Fill the **Service Attachment** which is configured in [Setup Self Hosted Kafka Private Service Connect in Google Cloud](/tidb-cloud/setup-self-hosted-kafka-psc.md)
+      4. Fill the **Boostrap Ports**, suggest provide more than one ports. You can use commas `,` to separate multiple ports.
+2. Select an **Authentication** option according to your Kafka authentication configuration.
     - If your Kafka does not require authentication, keep the default option **Disable**.
-    - If your Kafka requires authentication, select the corresponding authentication type, and then fill in the user name and password of your Kafka account for authentication.
+    - If your Kafka requires authentication, select the corresponding authentication type, and then fill in the **user name** and **password** of your Kafka account for authentication.
 
-3. Select your Kafka version. If you do not know that, use Kafka V2.
-4. Select a desired compression type for the data in this changefeed.
+3. Select your **Kafka Version**. If you do not know that, use Kafka V2.
+4. Select a desired **Compression** type for the data in this changefeed.
 5. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
-6. Click **Next** to check the configurations you set and go to the next page.
-
+6. Click **Validate Connection and Next** to test the network connection, if all is well it will go to the next page.
+> **Note:**
+>  If you select **Private Link** or **Private Service Connect** as network connectivity method. There will be extra steps compare to **Public IP** and **VPC Peering**.
+> 1. After you click the button, we will try to create endpoint in TiDB Cloud side for **Private Link** or **Private Service Connect**. It may take several minutes.
+> 2. After endpoint creation done, you need to accept the connection request in cloud vendor console with you account login.
+> 3. Then go back you TiDB Cloud console to confirm you have already accepted the connection request, then it will test connection and navigate to next page if everything OK.
```

Comment:
下文中的 click the button，是指哪个 button？
```suggestion

If you select **Private Link** or **Private Service Connect** as the network connectivity method, you need to perform the following extra steps.

1. After you click the button, TiDB Cloud creates the endpoint for **Private Link** or **Private Service Connect**. It might take several minutes.
2. After the endpoint is created, you need to log in the cloud vendor console and accept the connection request.
3. Go back your TiDB Cloud console to confirm that you have already accepted the connection request. TiDB Cloud will test the connection and navigate to the next page if everything works.
```